ansible_navigator/actions/replay.py

```diff
@@ -5,24 +5,22 @@
 from .run import Action as BaseAction
 
 
-class MissingAttribute:
-    # pylint: disable=too-few-public-methods
-    """Raise an attribute error for any get"""
-
-    def __get__(self, instance, owner):
-        raise AttributeError()
-
-
 @actions.register
-class Action(MissingAttribute, BaseAction):
+class Action(BaseAction):
 
     # pylint: disable=too-many-instance-attributes
     """:replay"""
 
+    @property
+    def mode(self):
```

Comment:
does this need to be here?  :run has a mode property, we should be able to use that I think

Reply:
The `mode` property in run becomes "stdout_w_artifact". That is _fine_ from _init_replay()'s workings since the if-conditions are branched with == "interactive" but for logging with the correct mode, we'd have to replace "stdout_w_artifact" with "stdout". But I'm okay with removing `mode` from replay.